frontend/src/lib/components/DebugNotice.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react'
+import { IconClose } from './icons'
+import { LemonButton } from './LemonButton'
+
+export function DebugNotice(): JSX.Element {
+    const [debugInfo, setDebugInfo] = useState<{ branch: string; revision: string } | undefined>()
+    const [noticeHidden, setNoticeHidden] = useState(false)
+
+    useEffect(() => {
+        const bottomNotice = document.getElementById('bottom-notice')
+        const bottomNoticeRevision = document.getElementById('bottom-notice-revision')?.textContent
+        const bottomNoticeBranch = document.getElementById('bottom-notice-branch')?.textContent
+
+        if (bottomNotice && bottomNoticeRevision && bottomNoticeBranch) {
+            setDebugInfo({
+                branch: bottomNoticeBranch || 'unknown',
+                revision: bottomNoticeRevision || 'unknown',
+            })
+
+            bottomNotice.remove()
```

Comment:
So now we have two notices. Why not just remove the other one?

Reply:
We leave it there for any screens without the sidebar (login, signup etc.)